test-data/everest/egg/everest/model/config_flow.yml

```diff
@@ -116,5 +111,5 @@ forward_model:
   - well_constraints  -i files/well_readydate.json -c files/wc_config.yml -rc well_rate.json -o wc_wells.json
   - add_templates     -i wc_wells.json -c files/at_config.yml -o at_wells.json
   - schmerge          -s eclipse/include/schedule/schedule.tmpl -i at_wells.json -o eclipse/include/schedule/schedule.sch
-  - myflow              r{{ eclbase }} --enable-tuning
```

Comment:
In this PR, support for arbitrary options forwarded to `flow` is not included (yet?)

Reply:
This is now fixed. But `--enable-tuning` was changed to `--enable-tuning=true` as the former was not supported by todays version of `flow`.